aica_docs/set_up_reinforcement_learning.md

```diff
@@ -211,9 +213,19 @@ class ReachSceneCfg(InteractiveSceneCfg):
         spawn=sim_utils.DomeLightCfg(color=(0.75, 0.75, 0.75), intensity=2500.0),
     )
 ```
+#### Defining the Observations
+
+The environment's **scene** represents its complete state, while **observations** represent the subset of this state 
```

Comment:
feels like here we also define what the scene is, but its corresponding section is the previous one. At the same time I wouldn't split it, perhaps we add a mini-sentence there, and here we say "While the environment's **scene** ..." and then what you have

Reply:
@bpapaspyros is this resolved for you?

Reply:
partially yes, my idea was to also have a sentence for this in the previous section otherwise my suggestions/current version doesn't work as well